### PR TITLE
Create ahmedraza.html

### PR DESCRIPTION
**Pull request by ahmedrazabaloch**
loading dot animation

### CODE REVIEW

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Thanks for this new contribution.

Please check the following checklist that you need to fulfill for us to be able to merge your request.

Please see the [Guide in README.md](https://github.com/zero-to-mastery/Animation-Nation#how-to-contribute) for further details.

##  Checklist
- [x] created a submission into the `./Art` folder
- [x] the contribution folder in `./Art` is the contributor's Github name
- [ ] has a gif or png file of the animation
- [x] can find the html 
- [x] can find the style (css file) **same file**
- [ ] modified `include.js` **with the right details**

Please refer to the `README.md` file for guidance and follow the guidelines.
If you have questions feel free to reach to the discord or ask them here.

**Comment by sophiabrandt** [comment]
@ahmedrazabaloch Hey, sorry, maybe I haven't checked that correctly, but there are still some problems with your pull request.


Please see the [Guide in README.md](https://github.com/zero-to-mastery/Animation-Nation#how-to-contribute) for further details.

##  Checklist
- [x] created a submission into the `./Art` folder
- [ ] the contribution folder in `./Art` is the contributor's Github name
- [ ] has a gif or png file of the animation
- [x] can find the html 
- [x] can find the style (css file or inline)
- [ ] modified `include.js` **with the right details**

## Issues: 

- the artwork is not in the correct folder
- gif or png missing
- after you have created the correct folder, you need to adjust `include.js`

Please refer to the `README.md` file for guidance and follow the guidelines.
If you have questions feel free to reach out on the discord channel #hacktoberfest-2023 or ask them here in this pull request.

File: include.js [hunk 1]
```diff
@@ -7,6 +7,12 @@ let cards = [
     githubLink: 'https://github.com/aniketmdinde'
   },
   {
+    artName 'Loading Dot Animation',
+    pageLink '.Artahmedrazaindex.html',
+    imageLink '.ArtahmedrazaLoading-Dot.gif',
+    author 'Ahmed raza',
+    githubLink 'httpsgithub.comahmedrazabaloch'
+ },
```
Comment (LaurelineP) [inline include.js:15]: # Feedback - Invalid JS Object
- [ ] missing colon (`:`) between the key and the value
- [ ] missing curly next curly braces for the object bellow
- [ ] incorrect paths/links for [ pageLink, imageLink, githubLink ] 
those seems to miss all slashes (`/`)
Please unsure those are correct before pushing your next commit(s)

## Context 
```js
 {
    artName 'Loading Dot Animation',
    pageLink '.Artahmedrazaindex.html',
    imageLink '.ArtahmedrazaLoading-Dot.gif',
    author 'Ahmed raza',
    githubLink 'httpsgithub.comahmedrazabaloch'
 },
   artName: 'LoopLamina',
```

File: Art/ahmedraza.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  }
+  45% {
+    transform: scale(1, 1);
+  }
+  85% {
+    transform: scale(0, 0);
+  }
+}
+
+    </style>
```
Comment (LaurelineP) [inline Art/ahmedraza.html:65]: # Feedback - modularisations
Your current code contains both HTML and CSS whereas the guidelines encourage
to use modularisations ( separating HTML and CSS in order to import the file )
- [ ] modularise your CSS into 
   make a file CSS dissociated from your HTML, and import this new CSS file
   into your HTML file.
   Here a resource for you to understand how it is done : [w3schools - html link](https://www.w3schools.com/tags/tag_link.asp)

**Review by LaurelineP — CHANGES_REQUESTED** [review]
# Feedbacks  
Please address all check list item not checked before we can accept your MR
## Code review
- [ ]  modularise your css _(stated in your code)_
- [ ]  invalid js objects _(stated in your code)_
---- 
## Acceptance criteria for a contribition
- [ ] the contribution includes a new folder named after the author github user name  in `./Art`
- [x] the contribution includes an HTML file
    - [ ] no CSS
    - [x] no JS
    - [x] no SVG
    - [ ] has a `<link>` tag to the CSS file
- [ ] the contribution includes a CSS file
- [ ] the contribution includes the animated file (gif) or picture (jpeg, jpg, png )
    _the pictural representation must not include the browser frame, just the animation_
- [ ] the contribution includes the correct contribution's details in `include.js`

If you need guidance , please refer to the `README.md` file.

**Comment by l-white** [comment]
Closing PR due to inactivity

**Pull request closed without merging**